### ISSUE TITLE
Zugelassende Arzneimittel-Auflistung des Bundesinstitut für Arzneimittel und Medizinprodukte (BfArM) as JSON

### ISSUE THREAD

**Issue by Lukas2112** [bundesAPI/sofortmassnahmen#16]
Ich könnte mir vorstellen, dass es ein leichtes sein könnte, die zugelassenden Arzneimittel des Bundesinstitut für Arzneimittel und Medizinprodukte (BfArM) als JSON bereitzustellen. Mit dieser API könnte man dann bspw. eine Webbasierte Anwendung bauen, die dann als Infoscreen in Apotheken und Arztpraxen ausgestrahlt werden könnten. 

Siehe: https://www.pharmnet-bund.de/static/de/index.html

**Comment by LilithWittmann** [comment]
Also zwei Optionen:
- Wir stellen eine Anfrage nnach 2. Open Data Gesetz
- Wir integrieren einen Scraper ins deutschland Paket (https://portal.dimdi.de/amguifree/am/search.xhtml)

**Comment by rflrkn** [comment]
Der Issue hier ist nu zwar schon beinahe vier Jahre alt, allerdings bin ich im Rahmen einer Projektidee auch über das Hindernis gestolpert, dass es scheinbar nach wie vor keine vollständige, öffentliche und freie Medikamentendatenbank in DE gibt. Ich wäre da also auch sehr dran interessiert. 

Einen Scraper zu basteln sieht für mich - wenn ich mir das Portal und die Suchmaske mal so ansehe - recht umständlich aus. 
Da ich keinerlei Erfahrungen damit habe: wie müsste denn eine Anfrage nach Open Data Gesetz aussehen, und hätte das deiner Meinung nach Aussicht auf Erfolg, @LilithWittmann? Gibt es da ähnliche Beispiele für, an denen man sich orientieren könnte?

**Comment by RaphaelWimmer** [comment]
Macht so eine formale Anfrage überhaupt Sinn? [Seit 13. Februar 2025 steht die Datenbank kostenfrei zur Verfügung](https://www.bfarm.de/DE/Arzneimittel/Arzneimittelinformationen/Arzneimittel-recherchieren/_node.html).

Das Kopieren der Datenbank ist ["aus Gründen der Aktualität nicht sinnvoll und deshalb nicht erlaubt"](https://www.bfarm.de/DE/Arzneimittel/Arzneimittelinformationen/Arzneimittel-recherchieren/AMIce/Datenbankinformation-AMIce-Arzneimittel/_node.html)

Ich würde fast vorschlagen, erstmal freundlich beim BfArM nachzufragen, ob sie eine REST-API bereitstellen werden/wollen, bzw. ob sie einen dabei unterstützen würden.

**Comment by rflrkn** [comment]
Ja, genau das habe ich gestern tatsächlich auch einfach mal gemacht. Im Falle einer Antwort werd ich hier ein Update geben: 

> Guten Tag,
> 
> Im Rahmen eines privaten, kleinen Softwareprojekts benötige ich Metainformationen zu allen in Deutschland zugelassenen Medikamenten. Diese Daten liegen allesamt im "AMIce - Öffentlicher Teil“ vor, und haben mir in kleinerem Umfang bereits weiterhelfen können. Für die eigentliche Umsetzung meines Projekts benötige ich eben jene Daten nun allerdings in maschinenlesbarer Form, idealerweise via API-Schnittstelle in JSON-Form o. ä.
> 
> Leider bietet die Rechercheoberfläche eine solche Schnittstelle meines Wissens nach nicht an - daher wäre hier nun meine Frage, ob die Bereitstellung einer solchen Schnittstelle nach Open Data-Prinzip (ggf. auch im Sinne des Open Data Gesetzes) angedacht ist, oder zukünftig zumindest denkbar wäre.
> 
> Ich freue mich über Ihre Antwort und verbleibe mit freundlichen Grüßen

**Comment by rflrkn** [comment]
> [...]
> 
> vielen Dank für Ihre Anfrage. Leider können wir derzeit keinen Komplettdownload bereitstellen, die Verfügbarkeit wird jedoch mittelfristig geprüft werden. Vermutlich haben Sie bereits festgestellt, dass der Export für bis zu 2000 Arzneimitteln gleichzeitig möglich ist.
> 
> Ergänzend möchten wir mitteilen, dass Arzneimitteldaten nicht der Open Data Definition unterliegen, da eine Voraussetzung dafür ist, dass die Daten keine Änderung erfahren.
> 
> Mit freundlichen Grüßen
> Im Auftrag
>
> [...]

Hm, das klingt ja erstmal nur so semi-vielversprechend, wenn wir mal ehrlich sind.

**Comment by marco927** [comment]
Also ich finde es echt traurig, dass es immer so langsam sein muss, wenn man mit deutschen Behörden zu tun hat. Die Aussage "...die Verfügbarkeit wird jedoch mittelfristig geprüft werden." bedeutet für mich da passiert vermutlich die nächsten 5 Jahre  erstmal gar nix.

Hoffe mal man hört doch schneller davon.
